sys-apps/firejail/firejail-0.9.64.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit toolchain-funcs python-single-r1
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/netblue30/${PN}/releases/download/${PV}/${P}.tar.xz"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/netblue30/firejail.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="Security sandbox for any type of processes"
+HOMEPAGE="https://firejail.wordpress.com/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="apparmor +chroot contrib +dbusproxy +file-transfer +globalcfg +network +overlayfs +private-home +suid test +userns vim-syntax +whitelist x11"
+RESTRICT="!test? ( test )"
+
+RDEPEND="apparmor? ( sys-libs/libapparmor )
+	contrib? ( ${PYTHON_DEPS} )
+	dbusproxy? ( sys-apps/xdg-dbus-proxy )"
+
+DEPEND="${RDEPEND}
+	!sys-apps/firejail-lts
+	sys-libs/libseccomp
+	test? ( dev-tcltk/expect )"
+
+REQUIRED_USE="contrib? ( ${PYTHON_REQUIRED_USE} )"
+
+src_prepare() {
+	default
+
+	find -type f -name Makefile.in | xargs sed -i -r \
+			-e '/^\tinstall .*COPYING /d' \
+			-e '/CFLAGS/s: (-O2|-ggdb) : :g' || die
+
+	sed -i -r -e '/CFLAGS/s: (-O2|-ggdb) : :g' ./src/common.mk.in || die
+
+	# remove compression of man pages
+	sed -i -e '/gzip -9n $$man; \\/d' Makefile.in || die
+	sed -i -e '/rm -f $$man.gz; \\/d' Makefile.in || die
+	sed -i -r -e 's|\*\.([[:digit:]])\) install -m 0644 \$\$man\.gz|\*\.\1\) install -m 0644 \$\$man|g' Makefile.in || die
```

Comment:
You can combine these modifications to a single `sed` call, since they edit the same file. I'd also prefer this as a patch, since it does so much. This solution should also be upstreamed.

Reply:
Combined.

I can create an upstream issue/PR. Is there some standard "precompressed manpages considered harmful" reference you know of that I can cite? This naive grep finds 68 instances of packages un-compressing existing manpages:

```
portage-tree $ find . -name \*.ebuild | xargs egrep '([bg]unzip|[bg]zip2? -d).*(man|\.[1-8])' | wc -l
68
```